README.md

```diff
@@ -85,6 +85,12 @@ npm install
 
 then
 
+```
+npm run generate
```

Comment:
I added this line to the setup phase as well since I was running into errors getting the FE to run without it